### PR TITLE
feat(@vtmn/svelte): add `VtmnTabs` component

### PR DESCRIPTION
**Pull request by Mig-l**
## Checklist

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Please, don't request directly from your main!
- [x] Check commits & PR names matches our requested structure. It must follow the https://www.conventionalcommits.org pattern.
- [x] Check your code additions will fail neither code linting checks.
- [x] I have reviewed the submitted code.
- [x] I have tested on related showcases.
- [x] If it includes design changes, please ask for a review with a core team designer.

## Does this introduce a breaking change?

- No

### CODE REVIEW

File: packages/showcases/svelte/stories/components/navigation/VtmnTabs/VtmnTabs.stories.svelte [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      </VtmnTabsItem>
+      <VtmnTabsItem>
+        Tab Name 2
+        <VtmnBadge value={6} />
+      </VtmnTabsItem>
+      <VtmnTabsItem>Tab Name 3</VtmnTabsItem>
+      <VtmnTabsItem>
+        <span aria-hidden="true" class="vtmx-heart-line" />
+        Tab Name 4
+        <VtmnBadge value={99} />
```
Comment (lauthieb) [inline packages/showcases/svelte/stories/components/navigation/VtmnTabs/VtmnTabs.stories.svelte:76]: As we did with `VtmnChip`, I propose you to add two props:
`icon` & `badgeValue`. With this, the slot will only be the label inside the tabs item.

https://github.com/Decathlon/vitamin-web/blob/5f592be743a51315182475d5424f36358e900fc9/packages/showcases/core/csf/components/selection-controls/chip.csf.js#L68-L90

**Review by lauthieb — CHANGES_REQUESTED** [review]

File: packages/showcases/svelte/stories/components/navigation/VtmnTabs/VtmnTabs.stories.svelte (hunk 1 above)
Comment (Mig-l) [reply packages/showcases/svelte/stories/components/navigation/VtmnTabs/VtmnTabs.stories.svelte]: This is a draft PR, I haven't finished

**Comment by Tlahey** [comment]
#1077 fix issue with icon colors

File: packages/sources/svelte/src/components/navigation/VtmnTabs/VtmnTabsItem.svelte [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<script>
+  import VtmnBadge from './../../indicators/VtmnBadge/VtmnBadge.svelte';
+  import VtmnIcon from './../../../guidelines/iconography/VtmnIcon/VtmnIcon.svelte';
+  export let href = '#';
+  export let badge = undefined;
```
Comment (Tlahey) [inline packages/sources/svelte/src/components/navigation/VtmnTabs/VtmnTabsItem.svelte:5]: Please change it to `badgeValue` like on `VtmnChip`

Comment (Mig-l) [reply packages/sources/svelte/src/components/navigation/VtmnTabs/VtmnTabsItem.svelte]: Done

**Review by Tlahey — DISMISSED** [review]

**Review by Tlahey — APPROVED** [review]

**Review by lauthieb — APPROVED** [review]

**Pull request merged**